grant-project/grant-monorepo
# Admin adjustments

### What this does
- removes logic from `admin DELETE` user & proposal end-points, simply returns a message with code 400
- explicit check for web3 connectivity, and reflect in state/UI

### Test
- try deleting user & proposal, should display a not implemented error message
- disable ganache OR disable MM and fiddle with the `localProviderString` in `web3helper.ts` to cause web3 calls to fail. See that for each proposal the web3 field shows 'UNAVAILABLE'.
